systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/ServiceUtils.java

```diff
@@ -76,12 +77,24 @@ public static void waitForNodePortService(String serviceName) throws Interrupted
      * Wait until Service of the given name will be deleted.
      * @param serviceName service name
      */
-    public static void waitForServiceDeletion(String serviceName) {
-        LOGGER.info("Waiting for Service {} deletion in namespace {}", serviceName, kubeClient().getNamespace());
+    public static void waitForServiceDeletion(String serviceName, String namespace) {
```

Comment:
Is there any other use case for this method?

Reply:
removed...